info.json

```diff
@@ -3,6 +3,7 @@
   "appBase": "https://app.wire.com",
   "bundleId": "com.wearezeta.zclient.mac",
   "copyright": "© Wire Swiss GmbH",
+  "customProtocolName": "wire",
```

Comment:
Any change to reuse `process.env.APP_CUSTOM_PROTOCOL_NAME` here?

Reply:
Not inside the default info JSON file but we override here if `APP_CUSTOM_PROTOCOL_NAME` env is set: https://github.com/wireapp/wire-desktop/pull/2145/files#diff-35b4a816e0441e6a375cd925af50752cR55

Reply:
That's great! So we can have custom links for custom apps by relying on `baseData.customProtocolName`.